_rules/image-no-text-0va7u6.md

```diff
@@ -10,8 +10,9 @@ accessibility_requirements:
     failed: not satisfied
     passed: further testing needed
     inapplicable: further testing needed
-  wcag20:1.4.9: # Images of Text (No Exception) (AA)
+  wcag20:1.4.9: # Images of Text (No Exception) (AAA)
     forConformance: true
+    secondary: true
```

Comment:
This name of "secondary" isn't final yet, and will probably change. This'll be easy to change though so I'm not going to worry about it for now.